steve-chae-multiple-choice-questions.md

```diff
@@ -0,0 +1,96 @@
+<!--
+	Correct answer is 2.
+	
+	1 - NgModule is the name of the Decorator used for any Angular Module.
+	I feel one can easily mistake between NgModule and AppModule
+	
+	3 - while it's possible that the root module can be anything, 
+	convention is to name it AppModule and that's what our slides say. 
+	This could be a good chance to bring it up that it's convention.
+	
+	(https://angular.io/docs/ts/latest/guide/appmodule.html)
+-->
+The name for the root module should be _________.:
+
+1. `NgModule`
+1. `AppModule`
+1. anything
+
+---
+
+<!--
+	Correct answer is 1.
+	
+	The root module is used exactly for bootstrapping and the boostrap 
+	property is specially available to the root module
+	
+	2, 3, 4 - are also available to other modules and not really special.
+	If students pick this maybe we didn't stress the purpose of the root
+	module strong enough
+	 
+
+-->
+Which of the following property is special to the root module?
+
+1. `bootstrap`
+1. `declarations`
+1. `providers`
+1. None. The root module is just like any other modules
+
+---
+
+<!--
+	Correct answer is 2.
+	
+	A Module's elements (as defined in `declrations`) are private by default and need to be "exported" to be public 
+	
+	1. -- declarations are used to "declare" existence of certain elements
+	inside the module. If the student picks this, it could be that the student
+	is confused because both `declarations` and `exports` contain a list
+	of the Module's elements
+	
+	3. -- only Services (associated with `providers` property) are public by
+	default. If the student picks this, it could be that the student is 
+	confused between Module's elements (private by default) and and Services
+	(public by default) 
+
+-->
+What property is used to make a Module's elements public?
```

Comment:
I think the use of the word "element" is because it's also Directives and Pipes that might need to be exported.

Unfortunately the Angular docs don't have clear terminology here. Perhaps the word "classes" instead of elements?
(https://angular.io/docs/ts/latest/cookbook/ngmodule-faq.html#!#q-what-to-declare)